docs/enhancements/20190815-zone-aware-routing.md

```diff
@@ -58,43 +58,52 @@ Arguably inter-zone network latency should also be better than cross zone.
 
 ## Proposal
 
-The idea here is to have controller part of ingress-nginx to (1) detect what zone its current pod is running in and (2) detect the zone for every endpoints it knows about.
-After that it will post that data as part of endpoints to Lua land. Then Lua balancer when picking an endpoint will try to pick zone-local endpoint first and
-if there is no zone-local endpoint then it will fallback to current behaviour.
+The idea here is to have the controller part of ingress-nginx
+(1) detect what zone its current pod is running in and
+(2) detect the zone for every endpoint it knows about.
+After that, it will post that data as part of endpoints to Lua land.
+Then Lua balancer when picking an endpoint will try to pick zone-local endpoint first and
+if there is no zone-local endpoint then it will fall back to current behaviour.
 
-This feature at least in the beginning should be optional since it is going to make it harder to reason about the load balancing and not everyone might want that.
+This feature at least, in the beginning, should be optional since it is going to make it harder to reason about the load balancing and not everyone might want that.
```

Comment:
```suggestion
Initially, this feature should be optional since it is going to make it harder to reason about the load balancing and not everyone might want that.
```